detekt.yml

```diff
@@ -2,21 +2,17 @@ build:
   weights:
     comments: 0   # missing comments should not fail the build
 
-test-pattern: # Configure exclusions for test sources
```

Comment:
`test-pattern` is not supported anymore with the latest version of Detekt.
https://arturbosch.github.io/detekt/configurations.html#path-filters--excludes--includes